### DISCUSSION TITLE
Eufy S380 with 2 of the S330 cameras - Using the setup does it stream via RTSP into Apple HomeKit?

### DISCUSSION THREAD

**Discussion by TabJo** [fuatakgun/eufy_security#1013] [Q&A]
I think I have this set up correctly. Although step # _“7- If you have installed RTSP Simple Server Add-On, please put its IP Address and Port into Integration Configuration page.”_ got me. Where can I find the RTSP simple server add-on’s IP address and port number? There was a step where I had to put 8556 as 8554 was being used by my wyze docker bridge and used 1936 instead of 1935. Even though in the log when I stop and restart it shows 8554. ??

I do see my 3 devices (the hub and my 2 cameras and I clicked the slider to turn on RTSP but unsure how to get the RTSP url or how to add to Apple HomeKit.

**Comment by malaywastaken** [comment]
hey did you ever figure it out?

**Reply by TabJo** [reply]
Nope. Returned them.
